server/routes/index.ts

```diff
@@ -72,10 +58,32 @@ export default function routes(service: Services): Router {
     }
 
     const report = await incidentReportingApi.getReportWithDetailsById(id)
+    const reportedBy = (await manageUsersApiClient.getNamedUser(systemToken, report.reportedBy))?.name
```

Comment:
Don't worry about the report page too much, but once this is merged I can also show the names of the staff involved.